frontend/src/features/workspace/WorkspacePage.tsx

```diff
@@ -103,6 +108,11 @@ export const WorkspacePage = (): JSX.Element => {
     topRef,
     createFormModalDisclosure,
   } = useWorkspaceForms()
+  const { user } = useUser()
+
+  const ROLLOUT_ANNOUNCEMENT_KEY = ROLLOUT_ANNOUNCEMENT_KEY_PREFIX + user?._id
```

Comment:
usememo, maybe also don't set a key if there is no id yet, so nothing gets saved in localstorage as `xxxxxxxxundefined`

Reply:
Okay, I updated to only show announcement if `useUser` loading is done. So the key can only be set once there's a userId since key is set when announcement is closed